### PR TITLE
Updates location for meeting recordings

### PR DESCRIPTION
**Pull request by mattmccleary**
The old YouTube link send people to a page with no recordings, and the entire archive appears to be gone.

This doc update reflects proposal here: https://github.com/open-telemetry/community/pull/1431/

### CODE REVIEW

**Comment by svrnm** [comment]
this PR has a dependency on https://github.com/open-telemetry/community/pull/1431 being closed. 

@open-telemetry/governance-committee when the community PR is approved, please also approve that this is the change we want.

**Comment by svrnm** [comment]
@mattmccleary the related community PR has been merged, can you verify that this can be merged now, thanks

File: hugo.yaml [hunk 1]
```diff
@@ -101,9 +101,9 @@ params:
         icon: fas fa-image
         desc: Check out the CNCF logo page for OpenTelemetry
       - name: Meeting Recordings
-        url: https://www.youtube.com/channel/UCHZDBZTIfdy94xMjMKz-_MA
+        url: https://docs.google.com/spreadsheets/d/1SYKfjYhZdm2Wh2Cl6KVQalKg_m4NhTPZqq-8SzEVO6s
         icon: fab fa-youtube
-        desc: Watch our meeting recordings on YouTube
+        desc: Watch our meeting recordings on Zoom cloud
```
Comment (chalin) [inline hugo.yaml:106]:
```suggestion
        desc: Watch our meeting recordings on Zoom Cloud
```

**Review by chalin — APPROVED** [review]
See inline comment for a minor copyedit.

I'll push the refcache update.

File: hugo.yaml [hunk 2]
```diff
@@ -101,9 +101,9 @@ params:
         icon: fas fa-image
         desc: Check out the CNCF logo page for OpenTelemetry
       - name: Meeting Recordings
-        url: https://www.youtube.com/channel/UCHZDBZTIfdy94xMjMKz-_MA
+        url: https://docs.google.com/spreadsheets/d/1SYKfjYhZdm2Wh2Cl6KVQalKg_m4NhTPZqq-8SzEVO6s
         icon: fab fa-youtube
```
Comment (svrnm) [inline hugo.yaml:105]:
```suggestion
        icon: fab fa-video-camera
```

**Pull request merged**